src/compiler/checker.ts

```diff
@@ -16881,31 +16864,24 @@ m2: ${(this.mapper2 as unknown as DebugTypeMapper).__debugToString().split("\n")
             }
         }
 
-        function attachDebugPrototypeIfDebug(mapper: TypeMapper): TypeMapper {
-            if (Debug.isDebugging) {
-                return Object.setPrototypeOf(mapper, DebugTypeMapper.prototype);
-            }
-            return mapper;

```

Comment:
Does this need to stay here and then reference `Debug.__debugTypeToString` for perf reasons? I don't know how often these are called, but I guess we have to access Debug anyway to check `isDebugging`.

Reply:
Sincerely hope this doesn’t matter